larpandoracontent/LArControlFlow/StitchingCosmicRayMergingTool.cc

```diff
@@ -554,6 +557,9 @@ void StitchingCosmicRayMergingTool::StitchPfos(const MasterAlgorithm *const pAlg
             continue;
         }
 
+    if ((stitchedLArTPCs.first == nullptr) || (stitchedLArTPCs.second == nullptr))
```

Comment:
Check indentation (might just be display on GitHub). Can also have `if (!stitchedLArTPCs.first || !stitchedLArTPCs.second)` if you prefer, but this is fine.